paper/src/main/resources/config.yml

```diff
@@ -403,7 +403,7 @@ hacks:
       - org.bukkit.event.painting
       - org.bukkit.event.vehicle
       - org.bukkit.event.weather
-      - org.bukkit.event.world
+      - org.bukkit.event.worldUUID
```

Comment:
Same as above, world -> worldUUID

Reply:
This seems to occur on every use of `world` in the config

Reply:
Good catch, I auto refactored a variable and it seems intellij inferred that this was part of a private field called "world".